pkg/antctl/antctl.go

```diff
@@ -413,6 +411,12 @@ $ antctl get podmulticaststats pod -n namespace`,
 							usage:     "Comma separated Antrea OVS flow table names or numbers",
 							shorthand: "T",
 						},
+						{
+							name:      "table-names-only",
+							usage:     "Print all Antrea OVS flow table names only",
+							shorthand: "b",
```

Comment:
maybe we don't need the shorthand for this flag? I don't see a connection between `--table-names-only` and `-b`

Reply:
Removed